react-design/my-app/src/App.tsx

```diff
@@ -0,0 +1,23 @@
+import RotatingBanner from './RotatingBanner';
+import Buttons from './Buttons';
+import './App.css';
+
+const items = [
+  'Aardvark',
+  'Bengal',
+  'Caterpillar',
+  'Dromedary',
+  'Elephant',
+  'Ferret',
+];
+
+function App() {
+  return (
+    <div>
+      <RotatingBanner items={items} />
+      <Buttons />
```

Comment:
The Buttons need to be part of the RotatingBanner. The entire thing should be one component, perhaps built from several subcomponents.